include/roboteam_utils/RobotCommands.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <roboteam_utils/Vector2.h>
+#include <roboteam_utils/Angle.h>
+
+#include <vector>
+
+namespace rtt {
+
+enum class KickType {
+    NORMAL, // Horizontal kicking
+    CHIP    // Diagonally in the air kicking
+};
+
+typedef struct RobotCommand {
+    unsigned int id = -1;           // [0,15] The id of robot
```

Comment:
```suggestion
    int id = -1;           // [0,15] The id of robot
```
Setting an unsigned int as negative 1 causes an overflow

Reply:
This technically should not matter, as a too low number for a robot ID is the same as a too high number. I will remove the `unsigned` keyword, because it only limits us. Being able to use negative numbers might show useful for testing purposes in the future. I'll also remove the default of -1, as this only brings confusion when you create an empty robot command